src/Stratis.Bitcoin.Features.PoA/Events/FedMemberKicked.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using Stratis.Bitcoin.EventBus;
+
+namespace Stratis.Bitcoin.Features.PoA.Events
+{
+    /// <summary>
+    /// Event that is executed when federation member is kicked.
+    /// </summary>
+    /// <seealso cref="Stratis.Bitcoin.EventBus.EventBase" />
+    public class FedMemberKicked : EventBase
```

Comment:
I sonder if we could just do an EventBase<T>?

Reply:
No, an event type must always describe itself, having a "generic event" is not a good idea to communicate what the event is if you pass that type around and anyway you want to subscribe to a specific event, having a simple EventBase for many events mean that a subscriber will receive lot of events it's not interested in and of course it doesn't know if it's something he have to consider or not.